pkg/execution/debounce/debounce.go

```diff
@@ -332,15 +332,17 @@ func (d debouncer) updateDebounce(ctx context.Context, di DebounceItem, fn innge
 	}
 	switch out {
 	case -1:
-		log.From(ctx).Warn().
-			Int64("status", out).
-			Msg(ErrDebounceInProgress.Error())
+		log.From(ctx).Warn().Msg(ErrDebounceInProgress.Error())
```

Comment:
if the issue will be resolved, we can remove this log.
it was only added for debugging purposes.